### PR TITLE
Add missing `_default_manager` symbol to generated through model

### LINKED ISSUE #1742: `ManyToManyField.through` no longer has `_default_manager` attribute
# Bug report

Following the call for testing in #1740, I tried `django-stubs` git master and ran into this issue.

For example if `MyModel` has relation `manytomany = models.ManyToManyField(OtherModel)`

## What's wrong

When accessing `MyModel.manytomany.through._default_manager`, I'm getting error:

> "type[MyModel_manytomany]" has no attribute "_default_manager"  [attr-defined]

`MyModel.manytomany.through.objects` works fine, however.

I can also reproduce this after these changes in django-stubs test suite:

```patch
diff --git a/tests/typecheck/fields/test_related.yml b/tests/typecheck/fields/test_related.yml
index 8f21557e..8d0ce289 100644
--- a/tests/typecheck/fields/test_related.yml
+++ b/tests/typecheck/fields/test_related.yml
@@ -337,6 +337,7 @@
         from myapp.models import User
         reveal_type(User().friends)  # N: Revealed type is "django.db.models.fields.related_descriptors.ManyRelatedManager[myapp.models.User]"
         reveal_type(User.friends.through.objects.get())  # N: Revealed type is "myapp.models.User_friends"
+        reveal_type(User.friends.through._default_manager)
         reveal_type(User.friends.through().from_user)  # N: Revealed type is "myapp.models.User"
         reveal_type(User.friends.through().from_user_id)  # N: Revealed type is "builtins.int"
         reveal_type(User.friends.through().to_user)  # N: Revealed type is "myapp.models.User"
```

```
E   pytest_mypy_plugins.utils.TypecheckAssertionError: Invalid output: 
E   Actual:
E     ...
E     main:4: error: "Type[User_friends]" has no attribute "_default_manager" (diff)
```

## System information

- OS: macOS 13.6
- `python` version: 3.11.5
- `django` version: 4.1.11
- `mypy` version: 1.5.1
- `django-stubs` version: (git master, commit 84a085c784ee68318d6fb190cbdda216d569fb43)
- `django-stubs-ext` version: 4.2.2

### PR DESCRIPTION
**Pull request by flaeppe**
# I have made things!

Adds `_default_manager` to plugin generated through model for many to many field

## Related issues

Closes: #1742

### CODE REVIEW

File: tests/typecheck/fields/test_related.yml [hunk 1]
```diff
@@ -1088,6 +1089,7 @@
 
         reveal_type(MyModel.auto_through.through)
         reveal_type(MyModel.auto_through.through.mymodel)
+        reveal_type(MyModel.auto_through.through._default_manager)
```
Comment (sobolevn) [inline tests/typecheck/fields/test_related.yml:1092]: What about other m2m fields?

Comment (flaeppe) [reply tests/typecheck/fields/test_related.yml:1092]: You mean in addition to `_default_manager`? Or which fields?

Comment (intgr) [reply tests/typecheck/fields/test_related.yml:1092]: Ping @sobolevn? This solves the issue I had.

**Review by sobolevn — COMMENTED** [review]
Oh, sorry. I missed this!

I was thinking about `other_again` and `custom_through` fields. Right now we only test `auto_through.through`

**Comment by flaeppe** [comment]
I think we touch `custom_through` enough already and the idea of `other_again` was to ensure we handle multiple many to many fields on the same model (had a bug regarding that while implementing). But I suppose we could assert similarly on `other_again`, not sure how much value it adds though.

**Review by sobolevn — APPROVED** [review]
Ok, then!

**Comment by sobolevn** [comment]
And thank you for your hard work on this project! :clap:

**Comment by flaeppe** [comment]
> And thank you for your hard work on this project! 👏

Well thank you too, for the work that you've been doing

**Pull request merged**